ZipAssetsPlugin.php

```diff
@@ -54,4 +54,11 @@ public function getDeveloperUrl()
     {
         return 'http://www.itmundi.nl';
     }
+
```

Comment:
Please add this code according to PSR-2 specs (spaces instead of tabs, indicate function visibility, add docblocks)